pkg/controller/migration/namespace_migration.go

```diff
@@ -248,6 +250,11 @@ func (m *CoreNamespaceMigration) Run(ctx context.Context, log logr.Logger) error
 	if err := m.deleteKubeSystemTypha(ctx); err != nil {
 		return fmt.Errorf("failed to delete kube-system typha Deployment: %s", err.Error())
 	}
+	log.V(1).Info("kube-system typha deployment deleted")
+	if err := m.deleteKubeSystemServiceEndPointCm(ctx); err != nil {
+		return fmt.Errorf("failed to delete kube-system k8sServicesEndpoint ConfigMap: %s", err.Error())
+	}
+	log.V(1).Info("kube-system services endpoint configmap deleted")
```

Comment:
Should probably only log this if we actually delete the configmap (a large number of clusters won't have this configmap, and for those we don't want to log anything)

Can just move the log within `deleteKubeSystemServiceEdnpoint`